src/app/home-page/home-logged-out/home.component.html

```diff
@@ -60,8 +60,9 @@ <h4 class="title caption text-center landing-message">
           platform used by the <a class="white-link" href="https://genomicsandhealth.org/" target="_blank">GA4GH</a> for sharing
           Docker-based tools described with either the
           <a class="white-link" href="http://www.commonwl.org/" target="_blank">Common Workflow Language (CWL)</a>, the
-          <a class="white-link" href="https://software.broadinstitute.org/wdl/" target="_blank">Workflow Description Language (WDL)</a>, or
-          <a class="white-link" href="https://www.nextflow.io/" target="_blank">Nextflow</a>
+          <a class="white-link" href="https://software.broadinstitute.org/wdl/" target="_blank">Workflow Description Language (WDL)</a>,
+          <a class="white-link" href="https://www.nextflow.io/" target="_blank">Nextflow</a>, or
+          <a class="white-link" href="https://training.galaxyproject.org/" target="_blank">Galaxy</a>
```

Comment:
I lean towards being usegalaxy.org or galaxyproject.org. The training project is... training. It's sort of like pointing on the previous line to nf-core instead of Nextflow (but that analogy isn't perfect). You decide.

Reply:
Your analogy seems to be in favor of training.galaxyproject.org instead.

Going to usegalaxy.org, shows search tools on the top left but doesn't really talk much about the language itself.  Which parallels nf-core instead of Nextflow (which explains what Nextflow is).

As for galaxyproject.org, I considered it briefly but felt that it was less informative than the training.galaxyproject.org

It doesn't matter to me much, feel free to change it if it bothers you.